tests/e2e/assets/1.0.0-proj/protractor.conf.js

```diff
@@ -17,15 +17,16 @@ exports.config = {
   jasmineNodeOpts: {
```

Comment:
Can you remove the changes to this file? It's true that they are just harmless whitespace, but they are also unnecessary so we shouldn't change it without a good reason.